Marlin/src/pins/stm32g0/pins_BTT_SKR_MINI_E3_V3_0.h

```diff
@@ -256,6 +256,47 @@
 
     #endif
 
+  #elif EITHER(BTT_MINI_12864_V1, FYSETC_MINI_12864_2_1)
```

Comment:
This can be simplified to just `FYSETC_MINI_12864_2_1` since that is defined when `BTT_MINI_12864_V1` is defined.